Resources/config/services/product-handler.xml

```diff
@@ -124,5 +124,11 @@
 
             <tag name="messenger.message_handler"/>
         </service>
+
+        <service id="Sulu\Bundle\SyliusConsumerBundle\Model\Product\Handler\Query\LoadProductVariantChannelPricingQueryHandler">
```

Comment:
@rs2487 you can let this as it is as its done before, but want to mention that this is bad practice in bundles using FCQN as ids. See https://symfony.com/doc/current/bundles/best_practices.html#services

> If the bundle defines services, they must be prefixed with the bundle alias. For example, AcmeBlogBundle services must be prefixed with acme_blog.

Reply:
I agree with you by 100 % but i want be consistent with the existing IDs